src/rum/instrumentation/DdRumReactNativeNavigationTracking.tsx

```diff
@@ -0,0 +1,56 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+
+import React from 'react'
+import { Navigation } from 'react-native-navigation';
+import { DdRum } from '../../foundation';
+
+/**
+* Provides RUM auto-instrumentation feature to track user interaction as RUM events.
+* For now we are only covering the "onPress" events.
```

Comment:
is this description valid? it seems we are covering any navigation event, not `onPress`.

maybe it is worth to add that this class is dedicated to the Wix library?

Reply:
I think this is from the `user interaction` class